components/image/slider/src/index.js

```diff
@@ -64,6 +64,11 @@ export default function ImageSlider({
     sliderOptions.initialSlide || 0
   )
   const slides = getSlides(images, linkFactory)
+  const hasSingleImage = Boolean(slides.length === 1)
```

Comment:
`slides.length === 1` is already a boolean.

Reply:
It was to give a little more semantics, but yes, it obviously returns a Boolean.
I'll change it, thank you!